flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/JobAutoScalerImpl.java

```diff
@@ -137,6 +145,12 @@ public boolean scale(FlinkResourceContext<? extends AbstractFlinkResource<?, ?>>
                     EventRecorder.Component.Operator,
                     e.getMessage());
             return false;
+        } finally {
+            if (evaluatedMetrics != null) {
+                LOG.debug("Storing evaluated metrics {}", evaluatedMetrics);
```

Comment:
I think either this or the logging of the evaluated metrics at line 120 should be removed, otherwise this is going to be crazy verbose